angular-chess/src/app/utils/board.utils.ts

```diff
@@ -768,4 +768,108 @@ export default class BoardUtils {
 
     return squaresToMove;
   }
+
+  public static getFen(board: Board): string {
+    return BoardUtils.getPieceFen(board) + " "
+      + BoardUtils.getMoveRightFen(board) + " "
+      + BoardUtils.getCastleRightFen(board) + " "
+      + BoardUtils.getEnPassantFen(board) + " "
+      + BoardUtils.getPlyFen(board) + " "
+      + BoardUtils.getMoveFen(board);
```

Comment:
1. You can change BoardUtils to this. I think it make it easier to read.
2. https://masteringjs.io/tutorials/fundamentals/string-concat; I like the array solution.
![grafik](https://user-images.githubusercontent.com/97945202/153930221-f6af7a61-6d79-4e5f-8b5b-7bb7d8ec9c2f.png)
3. Perhaps it better to create an extra class for this construct.
* Name of class: FenCreator; board as input of constructor
* and methods as part of FenCreator.